README.md

```diff
@@ -1,24 +1,215 @@
+This is an example of how you may give instructions on setting up your project locally.
 
-Things you may want to cover:
+Modify this file to match your project and remove sections that don't apply.
 
-* Ruby version
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
 
-* System dependencies
+OPTIONAL SECTIONS:
+- FAQ
 
-* Configuration
+After you're finished please remove all the comments and instructions!
+-->
 
-* Database creation
+<!-- TABLE OF CONTENTS -->
 
-* Database initialization
+# 📗 Table of Contents
 
-* How to run the test suite
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [📷 Video Recording](#video-record)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Services (job queues, cache servers, search engines, etc.)
+<!-- PROJECT DESCRIPTION -->
 
-* Deployment instructions
+# 📖 Coin Counter <a name="about-project"></a>
 
-* ...
+Instant Budget is a mobile web application that can helps you keep track of your daily financially transaction and manage your budget where you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>View Template Engine ".erb"</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby On Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Register a new user:** a new user can register by filling out a form with specific information.
+- **User login:** a registered user can log in to the app using his credential.
+- **Create Category:** a user can create a category for his expenses.
+- **Create transaction:** a user can create a transaction that belongs to a specific category.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://nurka.onrender.com/home)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📷 Video Recording <a name="video-record"></a>
```

Comment:
Good job on recording the presentation but I think there's something wrong with the audio because there's no sound in that presentation. Kindly re-record the presentation and please make sure the audio is working fine. 👍 

<img src="https://media3.giphy.com/media/t7tdQ6rcNcG4ppXjdM/giphy.gif"/>